### PR TITLE
Implement Structure Design Page (Lab Notebook)

### PR DESCRIPTION
**Pull request by IcedTea2K**
- Page is too long, so video is too big to upload
- References are not complete, but it should be merged for now. We can fix them again on Thursday, when reviewing with the write-up team

### CODE REVIEW

File: src/views/computational-simulation/StructureDesignView.vue [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+                </p>
+
+                <section class="py-2" v-for="(structure, i) in structures">
+                    <h3 class="text-base text-white"> {{ structure.label }} </h3>
```
Comment (daniel-panhead) [inline src/views/computational-simulation/StructureDesignView.vue]: I would use sectionStyle for the styling here just for consistency

File: src/views/computational-simulation/StructureDesignView.vue [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+const sectionStyle = 'text-subtitle-sm lg:text-subtitle text-white mt-6 mb-4'
+const structureHeadings = ['Highest RMSF (nm)', 'LOWEST RMSF (nm)', '95% CI  (nm)']
+const structures = [
+    {
+        label: "Structure (4-gon): CONTROL",
+        graphics : ["", "", ""],
+        values : [1.090856, 2.946035, 2.508811]
+    }, 
+    {
+        label: "Structure: 3-gon",
```
Comment (daniel-panhead) [inline src/views/computational-simulation/StructureDesignView.vue]: Do they want you to specify "x-gonal Bipyramid" like in the doc or is just "x-gon" fine?

File: src/views/computational-simulation/StructureDesignView.vue [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+                    This consistent approach enables a more structured comparison across various DNA structures. Our observations indicate that, for the most part, these structures maintain a similar level of stability. Our reference structure, which serves as a benchmark, displays certain interconnected segments. This interconnectedness is also evident, albeit to a reduced degree, in another structure called the 5-gon.
+                    A point of interest across these structures is the presence of a segment or edge that isn't as stable as the others. This unstable segment could be a pivotal factor in understanding why some DNA structures have the ability to connect or polymerize. Research by Lin et. al supports this notion, demonstrating that specific DNA structures can link together, forming chains that are illustrated in their provided figure (Figure X). 
+                </p>
+                <CaptionedGraphics>
```
Comment (daniel-panhead) [inline src/views/computational-simulation/StructureDesignView.vue]: Probably throw in a caption specifying that this is Figure X

File: src/views/computational-simulation/StructureDesignView.vue [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+                            \begin{align*} 
+                            g(d) = b &= r\theta \\
+                                     &= \frac{d}{2}\frac{\pi}{2} \\
+                                     &= \frac{d\pi}{4}
+                            \end{align*}" :display-mode="true"/>
+                    </div>
+                </section>
+
+                <section>
+                    <h3 :class="sectionStyle"> Axial Edge </h3>
```
Comment (daniel-panhead) [inline src/views/computational-simulation/StructureDesignView.vue]: Change to "Diameter to Number of Nucleotides"

File: src/views/computational-simulation/StructureDesignView.vue [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+                    </p>
+                    <div class="text-center">
+                        <Equation 
+                            formula="
+                            \begin{align*} 
+                                n | N \land N/n \equiv a\text{ mod } 2
+                            \end{align*}" :display-mode="true"/>
+                    </div>
+                    <p>
+                        for integer solutions of <Equation formula="a, b"/> to exist. It should be noted that, given <Equation formula="a \in \mathbb{Z}"/>, <Equation formula="N/n \equiv a\text{ mod }2 \Longrightarrow n | N"/>. However, <Equation formula="n|N"/>, while the parity equation developed above is less complicated to employ since only tiny adjustments to <Equation formula="a,b"/> would be necessary to obtain the closest even or odd number that satisfies the condition.
```
Comment (daniel-panhead) [inline src/views/computational-simulation/StructureDesignView.vue]: Add "is a difficult criteria to satisfy, " between the second `n|N` and `while`

**Review by daniel-panhead — CHANGES_REQUESTED** [review]
Mostly looks good, equations were definitely worth your pain and suffering

Only general nitpick I have is the spacing for images/graphics, make sure there's a proper amount of whitespace so it looks nice. I also changed the `CaptionedGraphics` component in another branch already to have smaller caption font size so it's easier to differentiate from the actual content text.

File: src/views/computational-simulation/StructureDesignView.vue (hunk 5 above)
Comment (IcedTea2K) [reply src/views/computational-simulation/StructureDesignView.vue]: Alright boss

File: src/views/computational-simulation/StructureDesignView.vue (hunk 4 above)
Comment (IcedTea2K) [reply src/views/computational-simulation/StructureDesignView.vue]: Done boss

File: src/views/computational-simulation/StructureDesignView.vue (hunk 2 above)
Comment (IcedTea2K) [reply src/views/computational-simulation/StructureDesignView.vue]: When I added the content it was only x-gon, I think. I'll fix it 🖖

File: src/views/computational-simulation/StructureDesignView.vue (hunk 1 above)
Comment (IcedTea2K) [reply src/views/computational-simulation/StructureDesignView.vue]: Gotcha gotcha

**Review by daniel-panhead — APPROVED** [review]
Changed some formatting for better spacings + responsiveness; merge if everything looks good

**Comment by daniel-panhead** [comment]
Side note - you can't put other block-level elements like ol in a p tag - ex. anything that's not a span, b, i, etc.

**Pull request merged**